_posts/2019-07-07-inception-pipelines-pt7.markdown

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title:  "Seeds of Inception - Part 7"
+date:   2019-07-07
+tags: AWS Organizations CloudTrail S3 KMS IAM
+author: Pete Yandell
+image: img/inception-pipelines/seed_germination.png
+---
+
+**Part 7 Global CloudTrail** - A practical example for auditing your AWS infrastructure
+
+<!-- markdownlint-disable MD033 -->
+<link rel = "stylesheet"
+   type = "text/css"
+   href = "{{ site.url }}/img/inception-pipelines/part-7/carousel.css" />
+
+![trail]({{ site.url }}/img/inception-pipelines/part-7/cover.jpg)
+
+Hi, I'm Pipeline Pete and I welcome you to "AWS things you should know about but don't". In this lesson we'll learn how to enable [AWS CloudTrail](https://aws.amazon.com/cloudtrail/) at the [AWS Organizations](https://aws.amazon.com/organizations/) level so that you never have to think about setting it up again (or live with that perpetual niggley fear that someone has tampered with it inside your account).
+
+Typically you'll see [AWS CloudTrail](https://aws.amazon.com/cloudtrail/) enabled within an account and recording events into an [S3 Bucket](https://aws.amazon.com/s3/). When it is done this way though, there is always a possibility that a rogue actor, with the right permissions, can tampered with it. However [AWS CloudTrail](https://aws.amazon.com/cloudtrail/) can also be enabled through [AWS Organizations](https://aws.amazon.com/organizations/). This means you can pick your Security/Audit account to receive **ALL** the trail events and the individual accounts don't get a chance to tamper with them as they never see the trail. You even get the additional benefit that any new accounts that get added to your [AWS Organizations](https://aws.amazon.com/organizations/) automatically start recording events into the bucket.
```

Comment:
`can tampered with it` should be `can tamper with it`

Reply:
Fixed